### ISSUE TITLE
No Documentation of Custom Templates ... oh the Irony ;-)

### ISSUE THREAD

**Issue by machineghost** [jashkenas/docco#275]
The Docco documentation mentions a `--template` command-line option, but that's pretty much the extent of the documentation of custom templates (at least that I was able to find on the Docco site or via Google).  I'd really like to to use a custom template with Docco, so I tried to figure them out on my own ... unsucessfully.

I started by copying `.npm/docco/0.6.3/package/resources/classic/docco.jst` (which may well be totally incorrect; I was just guessing) .  Then I tried running Docco with `--template` and my copy, but I either get:

```
binding.lstat(pathModule._makeLong(path), makeCallback(callback));
        ^
TypeError: path must be a string
```

(if I try `--template foo/myTemplateCopy.jst`) or:

```
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
             ^
Error: ENOENT, no such file or directory
 '"/home/me/foo/myTemplateCopy.jst"'
```

(if I try `--template "foo/myTemplateCopy.jst"`)

In other words, if I quote the template path, Docco complains about it being quoted (I know it's not complaining about the actual path because the file does exist at that path), and if I don't quote the path Docco complains about the path not being a string.

I'm sure there's something really stupid I'm doing, but without any documentation it's hard to figure out what that might be.

Could you please consider adding a little bit of documentation on how to use a custom template?  Or failing that could you at least provide an example command-line invocation of Docco that includes a custom template?
